### DISCUSSION TITLE
Configure Custom Nameservers for GitHub Pages Site?

### DISCUSSION THREAD

**Discussion by Tx0actical** [community/community#51590] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I want to add my GitHub pages [site](https://tx0actical.github.io) to my Cloudflare dashboard for analytics and security.

One of Clouflare's instructions includes changing nameservers to `ed.ns.cloudflare.com` and `lucy.ns.cloudflare.com`, after determining the registrar. Which I couldn't do using [ICANN's](https://lookup.icann.org/en) tool. Even if I could, there's little chance that there's any admin panel somewhere where I can log in and configure custom nameservers.

Is there any solution to this?

Thanks.

**Comment by yoannchaudet** [comment]
@Tx0actical If you want to use Cloudflare, you need your own custom domain and point it entirely at Cloudflare. There are a lot of registrars out there where you can buy a custom domains from.

https://porkbun.com is probably my favorite.

**Reply by Tx0actical** [reply]
@yoannchaudet Many thanks!
